redfish-core/lib/led.hpp

```diff
@@ -238,25 +239,56 @@ inline void
                                nlohmann::json& jsonInput)
 {
     BMCWEB_LOG_DEBUG << "Get LocationIndicatorActive";
-
     nlohmann::json& jsonIn = jsonInput["LocationIndicatorActive"];
-    dbus::utility::getAssociationEndPoints(
-        objPath + "/identifying",
-        [aResp, &jsonIn](const boost::system::error_code& ec,
-                         const dbus::utility::MapperEndPoints& endpoints) {
+
+    sdbusplus::asio::getProperty<
+        std::vector<std::tuple<std::string, std::string, std::string>>>(
+        *crow::connections::systemBus, "xyz.openbmc_project.Inventory.Manager",
+        objPath, "xyz.openbmc_project.Association.Definitions", "Associations",
```

Comment:
I don't understand why we need to get the Associations property of objectPath, because we have already configured associations [1] for all LEDs [2], so we can directly get the endpoint of 'objPath + "/identifying"'

[1] https://github.com/ibm-openbmc/openbmc/blob/1050/meta-ibm/recipes-phosphor/inventory/phosphor-inventory-manager/p10bmc/ibm%2Crainier-2u_associations.json#L335-L345

[2] https://github.com/openbmc/phosphor-dbus-interfaces/blob/master/yaml/xyz/openbmc_project/Inventory/Item/README.md#inventory-association

Reply:
"types":
{
                        "rType": "identified_by",
                        "fType": "identifying"

                    },
 why above types  are not present with the system object . 
 
 As "identifying" not present for the system, command failed to get the associations and so LocationIndicatorActive is null.

Reply:
So, we should be adding `identifying` to the system instead of changing these codes right?

Reply:
Yes, that will fix this issue.
But this code change is trying to handle it at runtime,,to handle this kind of things - '/identify_led_group' (in 1030) and '/identifying' (1050 and upstream)

Reply:
and I agree with the comments above,,so have marked the commit as draft, to relook.

Reply:
:+1:

Reply:
@Alpana07 

You can look at my comment in https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=454695

This is my comment.

```
host-bmc/host_pdr_handler.cpp
@@ -1557,7 +1557,7 @@ void HostPDRHandler::getPresentStateBySensorReadigs(
                     state == PLDM_STATE_SET_IDENTIFY_STATE_ASSERTED,
                     hostEffecterParser, mctpEid);
                 std::vector<std::tuple<std::string, std::string, std::string>>
                    associations{{"identify_led_group",           <=====
                                   "identify_inventory_object", ledGroupPath}};
```

If it is related to LED handling, I think the area to change is PLDM / 1050 (not bmcweb) - by changing "/led_identify_group" to "/identifying".